types/npmlog/index.d.ts

```diff
@@ -2,65 +2,81 @@
 // Project: https://github.com/npm/npmlog#readme
 // Definitions by: Daniel Schmidt <https://github.com/DanielMSchmidt>
 //                 Zhu Zijia <https://github.com/littlepiggy03>
+//                 Joseph Wynn <https://github.com/wildlyinaccurate>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.4
+// TypeScript Version: 3.4
 
-export enum LogLevels {
-    silly = "silly",
-    verbose = "verbose",
-    info = "info",
-    http = "http",
-    warn = "warn",
-    error = "error",
-}
+import { EventEmitter } from "events";
 
-export interface StyleObject {
-    fg?: string;
-    bg?: string;
-    bold?: boolean;
-    inverse?: boolean;
-    underline?: boolean;
+declare namespace npmlog {
+    // TODO: newStream, newGroup, setGaugeTemplate and setGaugeTemplateSet need to be added
+    interface Logger extends EventEmitter {
+        (): any;
 
-    bell?: boolean;
-}
+        level: string;
+        record: MessageObject[];
+        maxRecordSize: number;
+        prefixStyle: StyleObject;
+        headingStyle: StyleObject;
+        heading: string;
+        stream: any; // Defaults to process.stderr
 
-export interface MessageObject {
-    id: number;
-    level: string;
-    prefix: string;
-    message: string;
-    messageRaw: string;
-}
+        log(level: LogLevels | string, message: string, ...args: any[]): void;
+        log(level: LogLevels | string, prefix: string, message: string, ...args: any[]): void;
 
-// TODO: newStream, newGroup, setGaugeTemplate and setGaugeTemplateSet need to be added
+        silly(message: string, ...args: any[]): void;
```

Comment:
README of `npmlog` didn't mention the usage ignoring prefix. (https://github.com/npm/npmlog#basic-usage)

Screenshot as another proof: (already confirmed the version is newest)

![the screenshot](https://user-images.githubusercontent.com/22839136/58557449-04713280-8251-11e9-9e4e-e8cd5a18ee0c.png)